openedx/core/djangoapps/programs/management/commands/backpopulate_program_credentials.py

```diff
@@ -85,7 +86,8 @@ def handle(self, *args, **options):
 
     def _load_run_modes(self, user):
         """Find all run modes which are part of a program."""
-        programs = get_programs(user)
+        use_catalog = waffle.switch_is_active('get_programs_from_catalog')
+        programs = get_programs(user, use_catalog=use_catalog)
```

Comment:
In case you're wondering: this function returns a response adhering to the contract established by the programs service, which the unit tests are designed to cover. I will remove this in a later commit an update the unit tests to cover responses from the catalog service.